cmd/wallet-web/src/components/Vaults/AddVaultModal.vue

```diff
@@ -5,104 +5,56 @@
-            items-center
-            py-4
-            px-8
-            w-full
-            border-b-2 border-neutrals-thistle
-          "
-        >
-          <span class="text-lg font-bold text-neutrals-dark">
-            {{ t('Vaults.addVault') }}
-          </span>
-          <div>
-            <!-- TODO: use inline svg instead once https://github.com/trustbloc/edge-agent/issues/816 is fixed -->
-            <img
-              class="w-6 h-6 cursor-pointer"
-              src="@/assets/img/Icons-sm--close-icon.svg"
-              alt="Close Icon"
-              @click="closeModal"
-            />
-          </div>
-        </div>
-        <div class="flex items-center px-8 pt-10 w-full">
-          <input-field
-            v-model="vaultName"
-            :helper-message="t('Vaults.addHelperMessage')"
-            :label="t('Vaults.addlabel')"
-            :placeholder="t('Vaults.placeholderLabel')"
-            :value="vaultName"
-            type="text"
-            maxlength="42"
-            @update="receivedVaultName($event)"
+        <span class="text-lg font-bold text-neutrals-dark">
+          {{ t('Vaults.addVault') }}
+        </span>
+        <div>
+          <!-- TODO: use inline svg instead once https://github.com/trustbloc/edge-agent/issues/816 is fixed -->
+          <img
+            class="w-6 h-6 cursor-pointer"
+            src="@/assets/img/Icons-sm--close-icon.svg"
+            alt="Close Icon"
+            @click="closeModal"
           />
         </div>
-        <!--footer-->
-        <div
-          class="
-            md:flex-row
-            lg:flex-row
-            gap-4
-            justify-start
-            md:justify-between
-            lg:justify-between
-            items-center
-            px-5
-            md:px-8
-            lg:px-8
-            pt-4
-            pb-5
-            text-center
-            bg-neutrals-magnolia
-            rounded-b-2xl
-            flex flex-col
-            border-t border-0 border-neutrals-thistle
-          "
-        >
-          <button class="w-full md:w-auto lg:w-auto btn-outline" type="button" @click="closeModal">
-            {{ t('Vaults.cancel') }}
-          </button>
-          <button
-            id="deleteButton"
-            class="order-first md:order-last lg:order-last w-full md:w-auto lg:w-auto btn-primary"
-            type="button"
-            @click="addVault"
-          >
-            {{ t('Vaults.add') }}
-          </button>
-        </div>
       </div>
-    </div>
+      <div class="flex items-center px-8 pt-10 w-full">
+        <input-field
+          v-model="vaultName"
+          :helper-message="t('Vaults.addHelperMessage')"
+          :label="t('Vaults.addlabel')"
+          :placeholder="t('Vaults.placeholderLabel')"
+          :value="vaultName"
+          type="text"
+          maxlength="42"
+          @update="receivedVaultName($event)"
+        />
+      </div>
+    </template>
+    <!--button container-->
```

Comment:
Maybe we just keep these comments in the Modal itself and remove from here, the template name is self-explanatory. Besides, can we be consistent and use camel-case for comments like this, which defines a title of the section in code? `<!-- Buttons Container -->`